specifyweb/stored_queries/format.py

```diff
@@ -235,7 +236,9 @@ def aggregate(self, query: QueryConstruct,
         aggregated = blank_nulls(group_concat(formatted, separator, *order_by_expr))
 
 
-        return subquery.query.add_column(aggregated).limit(limit).label(None)
+        aggregator_label = f"aggregator_{self.aggregator_count}"
+        self.aggregator_count = (self.aggregator_count + 1) % 1000
```

Comment:
Hmm why modulo?

Reply:
😁 lol, just to prevent the number getting too ridiculously big.  But ya, since a new ObjectFormatter object is created for each query, we don't need this.